packages/blip/app/components/chart/stats.js

```diff
@@ -105,6 +106,20 @@ class Stats extends React.Component {
     }
     return stats.map(stat => {
       switch (stat.id) {
+        case CBGStatType.TimeInAuto:
+          return (
+            <div key={stat.id} data-testid={`stat-${stat.id}`}>
```

Comment:
I don't know to what extent we're willing to make changes to this file, but here the only thing the `switch/case` should compute is the `Stat` component based on the `CBGStatType`. In other words, I think it would make more sense to move the `switch/case` part returning only the appropriate component in a dedicated method, and here we could factorize what is computed above and below (`<div key=...>` and `<Divider>`).

This is what the result would look like here:

```js
return stats.map(stat => ({
  <div key={stat.id} data-testid={`stat-${stat.id}`}>
    {getStatElementById(stat.id)}
    <Divider variant="fullWidth" />
  </div>
}))
```